feature-libs/organization/order-approval/core/guards/approver.guard.ts

```diff
@@ -23,9 +23,11 @@ export class ApproverGuard implements CanActivate {
 
   canActivate(): Observable<boolean> {
     return this.userService.get().pipe(
-      filter((user: User) => user && Object.keys(user).length > 0),
+      filter((user: User | undefined) =>
+        Boolean(user && Object.keys(user).length > 0)
+      ),
       pluck('roles'),
-      map((roles: string[]) => {
+      map((roles) => {
```

Comment:
Can't `roles` have a type (and doesn't it need to have one)?

Reply:
type can be `unknown`, because of `pluck`